templates/procedures/cp-sanctions.md.tmpl

```diff
@@ -58,6 +58,4 @@ other retaliatory action as a consequence.
    within 24 hours. That notification will include 1) identifying the
    individual sanctioned, 2) the reason for the sanction, and 3) specific
    procedures for service or account restriction / revocation or other
-   disciplinary actions as required.
-
-[Warning Notice Template](ref/sanction-notice.pdf)
```

Comment:
This file is actually present in `ref` folder. @hesto2 I think the issue is that in the Policies app, we don't handle PDF references/attachments. Any thoughts on how we can leverage it? Perhaps link to https://github.com/JupiterOne/security-policy-templates/raw/master/templates/ref/sanction-notice.pdf?